components/footer/index.tsx

```diff
@@ -0,0 +1,29 @@
+import Link from "next/link";
+import {ArrowTopStyles, Content, Description, TextContainer, Title, Wrapper} from "./footer.styled";
+import {Container} from "../../styles/global.styled";
+import ArrowTop from "../../assets/icons/topArrow.svg";
+
+const Footer = () => {
+  return (
+    <Content>
+      <Container>
+        <Wrapper>
```

Comment:
just wonder if this complext structure could be simplified:
    <Content>
      <Container>
        <Wrapper>
is it possible to replace it with one wrapper instead of having 3?

Reply:
I  not understand why I made such a structure here